Mathlib/Data/Fintype/Basic.lean

```diff
@@ -1229,6 +1229,48 @@ theorem map_univ_val_equiv (e : α ≃ β) :
     map e univ.val = univ.val := by
   rw [←congr_arg Finset.val (Finset.map_univ_equiv e), Finset.map_val, Equiv.coe_toEmbedding]
 
+/-- For functions on finite sets, they are bijections iff they map universes into universes. -/
+@[simp]
+theorem bijective_iff_map_univ_eq_univ (f : α → β) :
+    Function.Bijective f ↔ map f (Finset.univ : Finset α).val = univ.val := by
+  constructor
+  {
```

Comment:
In Lean 4, we don't use curly braces to delimit code blocks anymore, instead as the style guide says

> When new goals arise as side conditions or steps, they are indented and preceded by a focusing dot · (inserted as \.); the dot is not indented.